gcc/rust/checks/errors/borrowck/ffi-polonius/src/gccrs_ffi.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2020-2023 Free Software Foundation, Inc.
```

Comment:
Should be bumped to 2024 (same for all copyright)

Reply:
Fixed by new commit